client/src/components/ui/bottomNav.tsx

```diff
@@ -0,0 +1,140 @@
+import Link from "next/link";
+import * as React from "react";
+import { useState } from "react";
+
+interface BottomNavIconProps extends React.SVGProps<SVGSVGElement> {
+  className?: string; // use to change color of the svg pic
+  alt?: string;
+}
+
+const BottomNavIcon1: React.FC<BottomNavIconProps> = ({
+  className,
+  ...props
+}) => {
+  return (
+    <svg
+      width={24}
+      height={24}
+      viewBox="0 0 24 24"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+      {...props}
+    >
+      <path
+        d="M8 6h13M8 12h13M8 18h13M3 6h.01M3 12h.01M3 18h.01"
+        stroke="currentColor"
+        strokeWidth={2}
+        strokeLinecap="round"
+        strokeLinejoin="round"
+      />
+    </svg>
+  );
+};
+
+const BottomNavIcon2: React.FC<BottomNavIconProps> = ({
+  className,
+  ...props
+}) => {
+  return (
+    <svg
+      width={24}
+      height={24}
+      viewBox="0 0 24 24"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+      {...props}
+    >
+      <path
+        d="M10 3H3v7h7V3zM21 3h-7v7h7V3zM21 14h-7v7h7v-7zM10 14H3v7h7v-7z"
+        stroke="currentColor"
+        strokeWidth={2}
+        strokeLinecap="round"
+        strokeLinejoin="round"
+      />
+    </svg>
+  );
+};
+
+const BottomNavIcon3: React.FC<BottomNavIconProps> = ({
+  className,
+  ...props
+}) => {
+  return (
+    <svg
+      width={24}
+      height={24}
+      viewBox="0 0 24 24"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+      {...props}
+    >
+      <path
+        d="M20 21v-2a4 4 0 00-4-4H8a4 4 0 00-4 4v2M12 11a4 4 0 100-8 4 4 0 000 8z"
+        stroke="currentColor"
+        strokeWidth={2}
+        strokeLinecap="round"
+        strokeLinejoin="round"
+      />
+    </svg>
+  );
+};
```

Comment:
Consider moving all `BottomNavIcon` components and interfaces to `/components/icons.tsx`, and rename from `BottomNavIcon` to something more descriptive (describe the actual icons).